pmd-java/src/main/resources/category/java/bestpractices.xml

```diff
@@ -1051,10 +1051,11 @@ String name,
           since="5.1"
           message="Position literals first in String comparisons for EqualsIgnoreCase"
           class="net.sourceforge.pmd.lang.java.rule.bestpractices.PositionLiteralsFirstInCaseInsensitiveComparisonsRule"
-          externalInfoUrl="${pmd.website.baseurl}/pmd_rules_java_bestpractices.html#positionliteralsfirstincaseinsensitivecomparisons">
+          externalInfoUrl="${pmd.website.baseurl}/pmd_rules_java_bestpractices.html#positionliteralsfirstincaseinsensitivecomparisons"
+          deprecated="true">
         <description>
 Position literals first in comparisons, if the second argument is null then NullPointerExceptions
-can be avoided, they will just return false.
+can be avoided, they will just return false. This rule is replaced by the more general LiteralsFirstInComparisons rule.
```

Comment:
We should directly link to the other rule `{% rule "LiteralsFirstInComparisons" %}` and use a new paragraph - I'll change that, when merging.